EngineBay.Blueprints.Tests/Bases/BaseTestWithDbContext.cs

```diff
@@ -0,0 +1,57 @@
+﻿namespace EngineBay.Blueprints.Tests
+{
+    using EngineBay.Persistence;
+    using Microsoft.EntityFrameworkCore;
+
+    public class BaseTestWithDbContext<TContext> : IDisposable
```

Comment:
Log a ticket for us to look if it is possible to share these base classes, they seem pretty generic